plugins/out_td/td_config.c

```diff
@@ -77,9 +77,6 @@ struct flb_td *td_config_init(struct flb_output_instance *ins)
             return NULL;
         }
     }
-    else {
-        ctx->region = FLB_TD_REGION_US;
-    }
```

Comment:
If `tmp` is guaranteed not to be `NULL`, whole if statement has to be removed.

Reply:
Have rebased from maste. This was fixed at a later commit